### DISCUSSION TITLE
How can I run build command in only the package that has been changed

### DISCUSSION THREAD

**Discussion by mxvsh** [vercel/turborepo#1912] [Help] [answered]
Right now when I run `pnnpm dev` it goes into each package in my monorepo and runs `dev` script which spawns more than 15 processes of node.js which eats up all my PC ram and eventually slowing down my PC.

How can I make it work like, "run build command only in the changed package"?

Thanks!

**Comment by gsoltis** [comment, marked as answer]
@xencodes have a look at [Filtering Workspaces](https://turborepo.org/docs/core-concepts/filtering) to filter the set of packages you're interested in.
